frontend/src/components/TopBar/ApplicationMenu.tsx

```diff
@@ -93,7 +106,13 @@ export default function ApplicationMenu(): JSX.Element {
         <MenuIcon />
       </Button>
       <Menu id='application-menu' anchorEl={anchorElement} open={Boolean(anchorElement)} onClose={closeMenu}>
-        <MenuItem onClick={loginClicked}>{t('topBar.menu.login')}</MenuItem>
+        {isAuthenticated ? (
+          <MenuItem onClick={logoutClicked}>Logout</MenuItem>
```

Comment:
The logout text should be internationalized to German and English. You can replace the `Logout` with `{t('topBar.menu.login')}` and then add the translations to `locales/de-global.json5` and `locales/en-global.json5`. The German translation would be `Abmelden`.